mock-app/index.html

```diff
@@ -0,0 +1,24 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Title</title>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <p>Conformance Framework Mock App</p>
+    <p>This app is only used by the conformance framework for test purposes.</p>
+    <p></p>
+    <p id="context"></p>
+    <script src="lib/fdc3-compliance.js"></script>
```

Comment:
The mock app doesn't need to include this script

Reply:
Thanks Colin, I've made that change